admin/scss/_style.scss

```diff
@@ -906,7 +906,7 @@ body.cms {
 			vertical-align: middle;
 		}
 
-		label {
+		span {
```

Comment:
Much better to use a class rather than element selector, then when markup changes we won't have to change our CSS again